### DISCUSSION TITLE
Switch built-in terminal to ghostyy?

### DISCUSSION THREAD

**Discussion by nilskch** [zed-industries/zed#22286] [Feature Requests]
I'm pretty excited about the upcoming release of ghostyy and @mitchellh talks a lot about the libghostyy library that enables you to embed ghostyy into other applications. I've seen maintainers not using the built-in terminal but instead using ghostyy in a separate application a lot.

I'm curious about two things:
1. Do you prefer Ghostyy over Alacrity as the built-in terminal?
2. How much work will it be to switch / what are the technical challenges here?

I know it's quite early to think about this since Ghostyy is not even publicly available yet and afaik libghostyy will not be stable at the initial release, but I'm curious what you think about this.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
